app/server/utils.py

```diff
@@ -405,6 +425,26 @@ def paint(self, documents):
         return res
 
 
+class SpacyPainter(JSONPainter):
+
+    def paint(self, documents):
+        data = super().paint(documents)
+        res = []
+        for d in data:
+            text = d['text']
+            annotations = d.pop('annotations')
+            entities = []
+            for a in annotations:
+                label_id = a["label"]
+                label_text = Label.objects.get(id=label_id).text
+                entities.append([a["start_offset"], a["end_offset"], label_text])
+            res.append( tuple([text, {"entities" : entities}]) )
```

Comment:
Nit: call to `tuple` is redundant and can be simplified to `res.append((text, {"entities" : entities}))`.